# Patch for 0.5.0 to fix problems and adjust configuration
# - Fix sbin dir in init script
diff -urNad xrdp-0.6.2-orig/instfiles/xrdp.sh xrdp-0.6.2/instfiles/xrdp.sh
--- xrdp-0.6.2-orig/instfiles/xrdp.sh	2016-10-04 08:53:24.000000000 +0300
+++ xrdp-0.6.2/instfiles/xrdp.sh	2020-04-19 14:06:49.846286140 +0300
@@ -5,7 +5,7 @@
 # chkconfig: 2345 11 89
 # description: starts xrdp
 
-SBINDIR=/usr/local/sbin
+SBINDIR=/usr/sbin
 LOG=/dev/null
 CFGDIR=/etc/xrdp
 
# - Adjust session manager configuration
diff -urNad xrdp-0.6.2-orig/sesman/sesman.ini xrdp-0.6.2/sesman/sesman.ini
--- xrdp-0.6.2-orig/sesman/sesman.ini	2016-10-04 08:53:24.000000000 +0300
+++ xrdp-0.6.2/sesman/sesman.ini	2020-04-19 14:06:49.886282793 +0300
@@ -6,7 +6,7 @@
 DefaultWindowManager=startwm.sh
 
 [Security]
-AllowRootLogin=1
+AllowRootLogin=0
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -20,7 +20,7 @@
 
 [Logging]
 LogFile=/var/log/xrdp-sesman.log
-LogLevel=DEBUG
+LogLevel=INFO
 EnableSyslog=0
 SyslogLevel=DEBUG
 
# - Fix make issue
diff -urNad xrdp-0.6.2-orig/sesman/tools/Makefile.am xrdp-0.6.2/sesman/tools/Makefile.am
--- xrdp-0.6.2-orig/sesman/tools/Makefile.am	2016-10-04 08:53:24.000000000 +0300
+++ xrdp-0.6.2/sesman/tools/Makefile.am	2020-04-19 14:06:49.953277186 +0300
@@ -20,7 +20,7 @@
 xrdp_sesrun_SOURCES = \
   sesrun.c \
   tcp.c \
-  config.c
+  ../config.c
 
 xrdp_sestest_SOURCES = \
   sestest.c
